app/src/main/java/org/simple/clinic/teleconsultlog/drugduration/DrugDurationModel.kt

```diff
@@ -2,7 +2,6 @@ package org.simple.clinic.teleconsultlog.drugduration
 
 import android.os.Parcelable
 import kotlinx.android.parcel.Parcelize
-import org.simple.clinic.teleconsultlog.drugduration.DrugDurationValidationResult.NOT_VALIDATED
 
 @Parcelize
 data class DrugDurationModel(
```

Comment:
We are not checking `hasDuration` computed property anymore since the pre-filled drug duration is getting set in the `Init`, can we remove that from the `DrugDurationModel` now?

Reply:
Good catch, removed it.